iree/compiler/Dialect/HAL/Target/ROCM/ROCMTarget.cpp

```diff
@@ -176,9 +169,17 @@ class ROCMTargetBackend final : public TargetBackend {
     iree_ROCMExecutableDef_start_as_root(builder);
 
     // Link module to Device Library
-    if (options_.ROCMLinkBC) {
-      LinkROCDLIfNecessary(llvmModule.get(), options_.ROCMTargetChip,
-                           options_.ROCMBitcodeDir);
+    if (clROCMLinkBC) {
+      LinkROCDLIfNecessary(llvmModule.get(), clROCMTargetChip,
+                           clROCMBitcodeDir);
+    }
+
+    //  Set amdgpu_hostcall if host calls have been linked, as needed by newer
+    //  LLVM
+    if (llvmModule->getFunction("__ockl_hostcall_internal")) {
```

Comment:
why would this function ever exist? Who would insert it? We wouldn't get host functions here.

Reply:
Sounds good! will remove :)